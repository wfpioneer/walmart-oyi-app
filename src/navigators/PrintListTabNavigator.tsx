import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
import React from 'react';
import { strings } from '../locales';
import PrintLists from '../screens/PrintList/PrintList';

const Tab = createMaterialTopTabNavigator();

export const PrintListTabNavigator = (): JSX.Element => (
  <Tab.Navigator>
    <Tab.Screen
      name="PriceSigns"
      options={{
        title: strings('PRINT.PRICE_SIGNS')
      }}
    >
      {() => <PrintLists tab="PRICESIGN" />}
    </Tab.Screen>
    <Tab.Screen
      name="Locations"
      options={{
        title: strings('PRINT.LOCATIONS')
      }}
    >
      {() => <PrintLists tab="LOCATION" />}
    </Tab.Screen>
  </Tab.Navigator>
);

const PrintListTabs = (): JSX.Element => (
  <PrintListTabNavigator />
);

export default PrintListTabs;
