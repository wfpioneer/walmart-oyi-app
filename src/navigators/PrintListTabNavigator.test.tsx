import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { PrintListTabNavigator } from './PrintListTabNavigator';

describe('PrintListTab Navigator', () => {
  it('Renders the PrintListTab navigator component', () => {
    const renderer = ShallowRenderer.createRenderer();
    renderer.render(
      <PrintListTabNavigator />
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
