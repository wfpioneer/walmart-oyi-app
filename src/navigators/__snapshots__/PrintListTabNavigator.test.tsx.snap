// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrintListTab Navigator Renders the PrintListTab navigator component 1`] = `
<MaterialTopTabNavigator>
  <Screen
    component={[Function]}
    name="PriceSigns"
    options={
      Object {
        "title": "Price Signs",
      }
    }
  />
  <Screen
    component={[Function]}
    name="Locations"
    options={
      Object {
        "title": "Locations",
      }
    }
  />
</MaterialTopTabNavigator>
`;
